core-plugins/src/test/java/co/cask/hydrator/plugin/transform/PythonEvaluatorTest.java

```diff
@@ -132,6 +132,31 @@ public void testSimple() throws Exception {
     Assert.assertEquals(expectedListField, output.get("arrayField"));
   }
 
+  @Test(expected = Exception.class)
+  public void testSchemaValidationInvalidScript() throws Exception {
+    Schema outputSchema = Schema.recordOf(
+      "smallerSchema",
+      Schema.Field.of("intField", Schema.of(Schema.Type.INT)),
+      Schema.Field.of("longField", Schema.of(Schema.Type.LONG)));
+
+    PythonEvaluator.Config config = new PythonEvaluator.Config(
+      "def transform(x, emitter, context):\n" +
+        " y = {}\n" +
```

Comment:
That's fine. Just that the test case was misleading (implied catching errors due to invalid schema).